apps/evm_family/avalanche/avalanche_app.c

```diff
@@ -0,0 +1,121 @@
+ * ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF
+ * CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+ * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *
+ *
+ * "Commons Clause" License Condition v1.0
+ *
+ * The Software is provided to you by the Licensor under the License,
+ * as defined below, subject to the following condition.
+ *
+ * Without limiting other conditions in the License, the grant of
+ * rights under the License will not include, and the License does not
+ * grant to you, the right to Sell the Software.
+ *
+ * For purposes of the foregoing, "Sell" means practicing any or all
+ * of the rights granted to you under the License to provide to third
+ * parties, for a fee or other consideration (including without
+ * limitation fees for hosting or consulting/ support services related
+ * to the Software), a product or service whose value derives, entirely
+ * or substantially, from the functionality of the Software. Any license
+ * notice or attribution required by the License must also include
+ * this Commons Clause License Condition notice.
+ *
+ * Software: All X1Wallet associated files.
+ * License: MIT
+ * Licensor: HODL TECH PTE LTD
+ *
+ ******************************************************************************
+ */
+
+/*****************************************************************************
+ * INCLUDES
+ *****************************************************************************/
+
+#include "avalanche_app.h"
+
+#include <stddef.h>
+
+/*****************************************************************************
+ * EXTERN VARIABLES
+ *****************************************************************************/
+
+/**
+ * @brief Whitelisted contracts with respective token symbol
+ * @details A map of Ethereum contract addresses with their token symbols. These
+ * will enable the device to verify the ERC20 token transaction in a
+ * user-friendly manner.
+ *
+ * @see erc20_contracts_t
+ */
+extern const erc20_contracts_t avalanche_contracts[];
+
+/*****************************************************************************
+ * PRIVATE MACROS AND DEFINES
+ *****************************************************************************/
+
+/*****************************************************************************
+ * PRIVATE TYPEDEFS
+ *****************************************************************************/
+
+/*****************************************************************************
+ * STATIC FUNCTION PROTOTYPES
+ *****************************************************************************/
+
+/*****************************************************************************
+ * STATIC VARIABLES
+ *****************************************************************************/
+
+static const evm_config_t avalanche_app = {
+    .lunit_name = "AVAX",
+    .name = "Avalanche CChain",
```

Comment:
Let's keep the exact name as previous unless there is a good enough reason to change.

Reply:
`#define EVM_LONG_NAME_MAX_SIZE 16`

Can we increase the max size?

Reply:
yes. definitely.

Reply:
do ensure you consider the null character when defining the size. and ensure to update the comment too.